sphinx/glossary.rst

```diff
@@ -540,6 +546,111 @@ Glossary
+
+      By default a suite server program is a `daemon`_ meaning that is runs in
+      the background (potentially on another host).
+
+   start
+   startup
+      When a :term:`suite` starts the cylc :term:`suite server program` is
+      run. This program controls the suite and is what we refer to as
+      "running".
+
+      * A :term:`Cylc suite` is started using ``cylc run``.
+      * A :term:`Rose suite configuration` (or :term:`Rosie Suite`) is started
+        using :ref:`command-rose-suite-run`.
+
+      A suite start can be either :term:`cold <cold start>` or :term:`warm <warm
+      start>` (cold by default).
+
+      See also:
+
+      * :ref:`Starting Suites`
+      * :term:`suite server program`
+      * :term:`warm start`
+      * :term:`cold start`
+      * :term:`shutdown`
+      * :term:`restart`
+      * :term:`reload`
+
+   cold start
+      A cold start is one in which the :term:`suite` :term:`starts <start>`
+      from the :term:`initial cycle point`. This is the default behaviour of
+      ``cylc run``.
+
+      See also:
+
+      * :term:`warm start`
+
+   warm start
+      In a :term:`datetime cycling` suite
+      a warm start is one in which the :term:`suite` :term:`starts <start>`
+      from a :term:`cycle point` after the :term`initial cycle point`.
+      Tasks in cycles before this point as assumed to have succeeded.
+
+      See also:
+
+      * :term:`cold start`
+
+   stop
+   shutdown
+      When a :term:`suite` is shutdown the :term:`suite server program` is
+      stopped. This means that no further :term:`jobs <job>` will be submitted.
+
+      By default cylc waits for any submitted or running :term:`jobs <job>` to
+      complete (either succeed or fail) before shutting down.
+
+      See also:
+
+      * :ref:`Stopping Suites`
+      * :term:`start`
+      * :term:`restart`
+      * :term:`reload`
+
+   restart
+      When a :term:`stopped <stop>` :term:`suite` is "restarted" Cylc will pick
+      up where it left off. Cylc will detect any :term:`jobs <job>` which
+      have changed state (e.g. succeed) during the period in which the
+      :term:`suite` was :term:`shutdown`.
+
+      See also:
+
+      * :ref:`Restarting Suites`
+      * :term:`start`
+      * :term:`Stop`
+      * :term:`Reload`
+
+   reload
+      Any changes made to the ``suite.rc`` file whilst the suite is running
+      will not have any effect untill the suite is:
+      
+      * :term:`Shutdown` and :term:`rerun <start>`
+      * :term:`Shutdown` and :term:`restarted <restart>`
+      * "Reloaded"
+
+      Reloading does not require the suite to be :term:`shutdown`. When a suite
+      is reloaded any currently "active" :term:`tasks <task>` will continue with
+      their "pre-reload" configuration, new tasks will use the new
+      configuration.
+
+      Reloading changes to the ``[runtime]`` section is safe, reloading changes
+      to the ``[scheduling]`` section has caveats, see the `Cylc User Guide`_.
```

Comment:
Note: `[runtime]` changes that affect the family inheritance hierarchy may affect scheduling as well.

Reply:
Oh, good point.

Reply:
See new commit.